dolly-backend-app/src/main/java/no/nav/dolly/bestilling/service/GjenopprettGruppeService.java

```diff
@@ -63,7 +83,34 @@ public void executeAsync(Bestilling bestilling) {
                                 List<Person> personer = tpsfService.hentTestpersoner(List.of(testident.getIdent()));
 
                                 if (!personer.isEmpty()) {
-                                    sendTestidenter(bestilling, bestKriterier, progress, personer.get(0));
+                                    TpsPerson tpsPerson = tpsfPersonCache.prepareTpsPersoner(personer.get(0));
+                                    sendIdenterTilTPS(new ArrayList<>(List.of(bestilling.getMiljoer().split(","))),
+                                            Stream.of(List.of(tpsPerson.getHovedperson()), tpsPerson.getPartnere(), tpsPerson.getBarn())
```

Comment:
Siden `List.of` allerede er en liste kan du droppe `Stream.of` heller bare skrive: 

`List.of(tpsPerson.getHovedperson()), tpsPerson.getPartnere(), tpsPerson.getBarn()).steam()`

Reply:
👍